experimental/scripts/configure_vscode.sh

```diff
@@ -0,0 +1,13 @@
+
+## Runs as user 
+
+install2.r languageserver lintr styler profvis httpgd
+sudo pip install radian
+
+## unfortunately these don't run and complete, but instead tries to bring up code-server
+#code-server install Ikuyadeu.r
+#code-server install RDebugger.r-debugger
+
+echo "options(vsc.use_httpgd = TRUE)" >> ~/.Rprofile
```

Comment:
The current vscode-R can describe this setting in `settings.json`, so it may be unnecessary.

Reply:
Ah, that's nice! I'm still very new to vscode-R -- where does that `settings.json` live?  Somewhere in the extensions dir presumably?  (as you can see in the note above, I haven't gotten `code-server` to install the extension automatically yet either...)

Reply:
The user's settings.json needs to be stored in the directory that is created when vscode is installed.
https://code.visualstudio.com/docs/getstarted/settings

Typically, users have their own settings.json that they will sync in some way (Microsoft's Visual Studio Code has a proprietary sync solution built in that will sync to GitHub account or Microsoft account).
I only use VisualStudio Code, so I don't know how to reuse settings.json in code-server in general (perhaps bindmount?).
https://coder.com/docs/code-server/latest/install#docker

```shell
mkdir -p ~/.config
docker run -it --name code-server -p 127.0.0.1:8080:8080 \
  -v "$HOME/.config:/home/coder/.config" \
  -v "$PWD:/home/coder/project" \
  -u "$(id -u):$(id -g)" \
  -e "DOCKER_USER=$USER" \
  codercom/code-server:latest
```